templates/cli/lib/commands/push.js.twig

```diff
@@ -1410,6 +1427,7 @@ push
     .command("collection")
     .alias("collections")
     .description("Push collections in the current project.")
+    .option(`-d, --debounce <maxDebounces>`, `Max number of debounces, default: 30.`)
```

Comment:
This description is pretty confusing for people because they don't know how the push code works. How can we rephrase this to be clearer?

Reply:
Is this better?